seacrowd/sea_datasets/xm3600/xm3600.py

```diff
@@ -0,0 +1,195 @@
+                    },
+                }
+            )
+        elif self.config.schema == "seacrowd_imtext":
+            features = schemas.image_text_features()
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        captions_path = dl_manager.download_and_extract(_URLS["captions"])
+        images_path = dl_manager.download_and_extract(_URLS["images"])
+        attr_path = dl_manager.download(_URLS["image_attributions"])
+
+        train_caps = {}
+        test_caps = {}
+        val_caps = {}
+
+        print(self.config.subset_id.split("_"))
+        current_lang = self.config.subset_id.split("_")[1]
+
+        img_df = pd.read_csv(attr_path)
+
+        img_df_train = img_df.loc[img_df["Subset"] == "train"][["ImageID", "Subset"]]
+        img_df_test = img_df.loc[img_df["Subset"] == "test"][["ImageID", "Subset"]]
+        img_df_val = img_df.loc[img_df["Subset"] == "validation"][["ImageID", "Subset"]]
+
+        with jl.open(os.path.join(captions_path, "captions.jsonl"), mode="r") as jsonl_file:
+            for line in jsonl_file:
+                if line["image/key"] in img_df_train.ImageID.values:
+                    train_caps[line["image/key"]] = line[current_lang]
+                elif line["image/key"] in img_df_test.ImageID.values:
+                    test_caps[line["image/key"]] = line[current_lang]
+                elif line["image/key"] in img_df_val.ImageID.values:
+                    val_caps[line["image/key"]] = line[current_lang]
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": {"img_ids": img_df_train.ImageID.values, "images": {img_id: os.path.join(images_path, img_id + ".jpg") for img_id in img_df_train.ImageID.values}, "captions": train_caps},
+                    "split": "train",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "filepath": {"img_ids": img_df_test.ImageID.values, "images": {img_id: os.path.join(images_path, img_id + ".jpg") for img_id in img_df_test.ImageID.values}, "captions": test_caps},
+                    "split": "test",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={
+                    "filepath": {"img_ids": img_df_val.ImageID.values, "images": {img_id: os.path.join(images_path, img_id + ".jpg") for img_id in img_df_val.ImageID.values}, "captions": val_caps},
+                    "split": "dev",
```

Comment:
remove `split` in `gen_kwargs`
```suggestion
            datasets.SplitGenerator(
                name=datasets.Split.TRAIN,
                gen_kwargs={
                    "filepath": {"img_ids": img_df_train.ImageID.values, "images": {img_id: os.path.join(images_path, img_id + ".jpg") for img_id in img_df_train.ImageID.values}, "captions": train_caps}
                },
            ),
            datasets.SplitGenerator(
                name=datasets.Split.TEST,
                gen_kwargs={
                    "filepath": {"img_ids": img_df_test.ImageID.values, "images": {img_id: os.path.join(images_path, img_id + ".jpg") for img_id in img_df_test.ImageID.values}, "captions": test_caps}
                },
            ),
            datasets.SplitGenerator(
                name=datasets.Split.VALIDATION,
                gen_kwargs={
                    "filepath": {"img_ids": img_df_val.ImageID.values, "images": {img_id: os.path.join(images_path, img_id + ".jpg") for img_id in img_df_val.ImageID.values}, "captions": val_caps}
```